marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-custom-writer.json

```diff
@@ -0,0 +1,4 @@
+{
+  "role-name": "data-hub-custom-writer",
```

Comment:
Pinging @ryanjdew  - I think it's fine to do this for now, but before 5.3 goes out, I lean towards renaming all of these to be "data-hub-(stepType)-step-writer" for clarity.

Reply:
I can get behind that. We need to should have a separate ticket for that.